external/owee/dune

```diff
@@ -7,7 +7,6 @@
    (:include %{project_root}/sharedlib_cflags.sexp)
    (:include %{project_root}/oc_cppflags.sexp)))
 )
```

Comment:
(Possibly add `(implicit_transitive_deps false)`.)

Reply:
(That should ensure the remove dependencies are not
picked up by accident.)

Reply:
Let's discuss separately.